torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -927,6 +927,11 @@ def _run_param_init_fn():
         # For validating execution order across ranks
         self._exec_order_data = _ExecOrderData()
 
+        # setting communication hook to a default
+        self.communication_hook = self._get_default_comm_hook()
+        self.communication_hook_state = self._get_default_comm_hook_state()
+        self._comm_hook_was_called = False
```

Comment:
nit: comm_hook_was_registered might be a better name?

Reply:
changed to "_hook_registered"